code/game/machinery/computer/robot.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "Used to remotely lockdown or detonate linked Cyborgs and Drones."
```

Comment:
If detonation is being changed to a malf ability, needs to have description updated.

Reply:
True, my bad. Will fix it in a moment